OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs

```diff
@@ -241,7 +244,7 @@ public IEnumerable<IRenderable> RenderAboveShroud(WorldRenderer wr, World world)
 				var palette = wr.Palette(power.Info.IconPalette);
 
 				// Destination tiles
-				foreach (var t in world.Map.FindTilesInCircle(xy, range))
+				foreach (var t in world.Map.FindTilesInCircle(xy, range, true))
```

Comment:
Unfortunately this only works for the invisible cordon where there are still backing map cells.  See the map Desert Rats for an example where there is only a 1 cell cordon.

Reply:
Grrrr... Alright, I'll look into that one more